azure-devops/azext_devops/dev/common/config.py

```diff
@@ -42,6 +44,14 @@ def set_global_config_value(section, option, value):
     azdevops_config.config_parser.read(AZ_DEVOPS_GLOBAL_CONFIG_PATH)
 
 
+def get_preview_value():
```

Comment:
Should have caching since this will affect -h commands which people expect to be fast.